Common/TableProducer/trackselection.cxx

```diff
@@ -53,7 +57,8 @@ struct TrackSelectionTask {
   {
     for (auto& track : tracks) {
       filterTable((uint8_t)globalTracks.IsSelected(track),
```

Comment:
Since with your changes every cut is checked for every track, we could also derive the IsGlobalTrack decision from the flag that you create below, right? This would avoid doing the checks multiple times.
IsGlobalTrack requires all of the bits to be set, so one could just check if it is equal to `uint8_t(~uint8_t(0))`. 
Then in the future one can also easily define additional 'filterBitmasks' / decision columns where for instance the dca or pt cuts are omitted.

Reply:
Indeed one could define the global quality as a dynamic column! Good point Mario, thanks!

Reply:
@mario-krueger what do you think of this? https://github.com/AliceO2Group/O2Physics/pull/691/commits/6a364353d9a8f8ddce666ffd94268d12f8356765 Unfortunately to handle the SSD one still needs a separate bool

Reply:
I like it :)